map-of-the-web/motw/motw/producing.py

```diff
@@ -0,0 +1,132 @@
+from bokeh.plotting import figure, output_file, show
+from bokeh.io import save
+from bokeh.layouts import gridplot 
+from motw.snapshot import Snapshot
+import requests as req
+from datetime import datetime
+import pandas as pd
+
+def timestamp2date(x):
```

Comment:
This looks like a helpful utility function; it probably belongs in some kind of util.py (or whatever the standard for django is)